configure

```diff
@@ -163,6 +164,17 @@ do
     esac
 done
 
+# if the user didn't explicitly specify whether they want docs
+# or not, enable them if a2x is found, otherwise disable.
+if [ -z "$enable_doc" ]; then
+    if command -v a2x >/dev/null 2>&1; then
+        enable_doc="true"
+    else
+        printf 'Warning: a2x was not found, documentation will not be built!\n' >&2
+        enable_doc="false"
+    fi
+fi
```

Comment:
If I forget to install asciidoc and run configure & make as usual, the toolchain will silently make a tarball that does not contain documentation, which is not acceptable.

Reply:
Fair, I made it error if documentation is enabled but a2x isn't found.